src/settings/VoucherNumber/SettingsVoucherNumberForm.js

```diff
@@ -80,7 +81,8 @@ const SettingsVoucherNumberForm = ({ onReset, firstSequenceNumber }) => (
 
 SettingsVoucherNumberForm.propTypes = {
   onReset: PropTypes.func.isRequired,
-  firstSequenceNumber: PropTypes.string.isRequired,
+  onChangeStartNumber: PropTypes.func.isRequired,
+  sequenceNumber: PropTypes.string.isRequired,
```

Comment:
why it's a string?

Reply:
Because BE return string.

Reply:
Also I don't like how you're treating default values. It should be done in one place and only when value for sequence number is empty on the back-end. However, please check with back-end guys, because I suppose they handle default value case already there.

Reply:
BE always waiting for value from us

Reply:
`sequenceNumber` is the value , fetched by the GET API. I don't think back-end is waiting anything from UI